README.md

```diff
@@ -23,15 +25,31 @@ Using the primitives shown in these scripts one could have a fleet of devices an
 ```
 Note that you need to provide the full commit hash for this command. It should also be noted that any devices that join/provision into the fleet after this is set, will download and update this commit. To confirm that this has worked, check the top right of the device list page and you should see the `Application commit:` value has been set to the commit you passed it in the command.
 
-4. If we want to have a few devices that are part of a testing group, we can use the `set-device-to-a-build.sh` script. For convenience, I defined an device env var `TEST` that is used to designate that this device is part of the testing group. So any device in my fleet that has the env var `TEST` can then be set with a testing build. The script `update-test-group.sh` will set all the devices with this env var to a specific commit/build.
+4. If we want to have a few devices that are part of a testing group, we can use the `set-device-to-a-build.sh` script:
+```
+./set-device-to-a-build.sh <DEVICE FULL UUID HERE> <FULL COMMIT HASH HERE>
+```
+
+5. For convenience, we can define an device env var `TEST` that is used to designate that this device is part of the testing group. So any device in the fleet that has the env var `TEST` can then be set with a testing build. The `update-test-group.sh` script will set all the devices with this env var to a specific commit/build:
```

Comment:
@thgreasi we should actually now use Tags instead of an Env Var, using the ENV var was just a hack, it was always designed to be used with tags :)

Reply:
@shaunmulligan I guessed so. While writing this, I stopped and gave a second and a third thought on whether I should rewrite that right away but in the end I ended up that it should be a separate PR :)
Opened #14 for this.

Reply:
@thgreasi yup definitely a separate PR